controller/space_areas.go

```diff
@@ -46,7 +47,7 @@ func (c *SpaceAreasController) List(ctx *app.ListSpaceAreasContext) error {
 			return jsonapi.JSONErrorResponse(ctx, err)
 		}
 		for _, a := range areas {
-			logrus.Info("Found space area with ID=", a.ID.String())
+			log.Info(ctx, map[string]interface{}{"area_id": a.ID}, "Found space area with id %s", a.ID)
```

Comment:
there's something that would need some clarification for me: should we customize the message with some values such as you do here (`"Found space area with id %s", a.ID`) or should we have a generic message and rely on the `map[string]interface{}{"area_id": a.ID}` arg to see the contextual values ? My underlying question being: are we going to collect/aggregate the logs by message at some point in the future ? If so, shouldn't we use generic messages ? Just asking for clarification.